### ISSUE TITLE
The 'previous' function isn't quite working well

### ISSUE THREAD

**Issue by CardinalSparrow** [CardinalSparrow/Quiz-app#1]
When the previous button is clicked, it should take you back to how things were but instead, it only takes you to the previous question while still maintaining the score. Thus, if you get the previous question correct twice, you have a score that is greater than the total score.

**Comment by Duesenberg** [comment]
Hello,

I came across your repository by chance and I really liked your quiz. I think I found a solution to your bug.

Basically added a new property to each of your questions in quizData, 'correctAnswer', which should have a value of either true or false.

Changed the logic in the checkAnswer function to change the 'correctAnswer' to true/false whether the selected answer is correct/incorrect.

Then added a new function calculateScore which goes through quizData at the end of the quiz and adds +1 to score if correctAnswer is true for each of the questions. This function gets called in endQuiz.

Took the quiz and works as intended. If you want, I can submit a pull request and add this!
